tutorials/ag2-multi-agent/README.md

```
@@ -0,0 +1,45 @@
+![](https://europe-west1-atp-views-tracker.cloudfunctions.net/working-analytics?notebook=tutorials--ag2-multi-agent--readme)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)